sdk/test-utils/recorder/README.md

```diff
@@ -1,18 +1,510 @@
+
+The following commands run the tests with the default configs and concurrently starts(runs) the test-proxy tool in a detached process in the background in record/playback modes if it is not already active. Additionally, more options can be passed to override the default configs.
+
+- `dev-tool run test:node-js-input -- --timeout 5000000 'dist-esm/test/**/*.spec.js'`
+- `dev-tool run test:node-ts-input -- --timeout 1200000 --exclude 'test/**/browser/*.spec.ts' 'test/**/*.spec.ts'`
+- `dev-tool run test:browser`
+  Read more at [dev-tool commands #usage](https://github.com/Azure/azure-sdk-for-js/blob/main/common/tools/dev-tool/README.md#usage)
+
+The test-proxy tool is run at ports 5000(for HTTP) and 5001(for HTTPS) unless you specify `TEST_PROXY_HTTP_PORT` as an environment variable, in which case that will be picked.
+
+Test scripts
+
+```json
+{
+  // ... your package.json scripts section
+  "integration-test:node": "...",
+  "unit-test:node": "..."
+  // ... more of your package.json scripts
+}
+```
+
+Your test scripts (in `package.json`) should be based on the following examples:
+
+| script name                | command                                                                                                          |
+| :------------------------- | :--------------------------------------------------------------------------------------------------------------- |
+| `unit-test:browser`        | `dev-tool run test:browser`                                                                                      |
+| `unit-test:node`           | `dev-tool run test:node-ts-input -- --timeout 1200000 --exclude 'test/**/browser/*.spec.ts' 'test/**/*.spec.ts'` |
+| `integration-test:browser` | `dev-tool run test:browser`                                                                                      |
+| `integration-test:node`    | `dev-tool run test:node-js-input -- --timeout 5000000 'dist-esm/test/**/*.spec.js'`                              |
+
+Note the difference between the dev-tool `node-ts-input` and `node-js-input` commands:
+
+- `node-ts-input` runs the tests using `ts-node`, without code coverage.
+- `node-js-input` runs the tests using the built JavaScript output, and generates coverage reporting using `nyc`.
+
+Read more at [dev-tool commands #usage](https://github.com/Azure/azure-sdk-for-js/blob/main/common/tools/dev-tool/README.md#usage)
+
+#### Prerequisites
+
+- [Docker](https://docker.com/) is required, as the [test proxy server](https://github.com/Azure/azure-sdk-tools/tree/main/tools/test-proxy) is run in a container during testing. When running the tests, ensure the Docker daemon is running and you have permission to use it. For WSL 2, running `sudo service docker start` and `sudo usermod -aG docker $USER` should be sufficient.
```

Comment:
I think it's worth adding something here with the installation link to Docker for windows. When I was reading this, I assumed I needed to use WSL.

Reply:
updated :)